src/CommandBasedBackendRegisterAccessor.cpp

```diff
@@ -44,7 +44,7 @@ namespace ChimeraTK {
 
     // Allocate the buffers
     NDRegisterAccessor<UserType>::buffer_2D.resize(_registerInfo.getNumberOfChannels());
-    NDRegisterAccessor<UserType>::buffer_2D[0].resize(_registerInfo.getNumberOfElements());
+    NDRegisterAccessor<UserType>::buffer_2D[0].resize(_numberOfElements);
```

Comment:
We should clearly document the difference in meaning between CommandBasedBackendRegisterAccessor::_numberOfElements and the CommandBasedRegisterInfo::getNumberOfElements, since these are no longer synonymous and the naming is now confusing.